test/client_service_base.test.js

```diff
@@ -567,6 +567,8 @@ describe('executeContract', () => {
       setCertVersion: function() {},
       setFunctionArgument: function() {},
       setSignature: function() {},
+      setUseFunctionIds: function() {},
```

Comment:
the related test will be added later in another PR after ClientService is revised.
Here it just makes the original tests pass.